src/features/attributeSelect.tsx

```diff
@@ -0,0 +1,379 @@
+            onClose();
+          }}
+        />
+      </div>
+    </Dialog>
+  );
+};
+const AttributeButton = ({
+  attributeName,
+  uid,
+}: {
+  attributeName: string;
+  uid: string;
+}) => {
+  return (
+    <Button
+      icon="chevron-down"
+      intent="primary"
+      minimal
+      onClick={(e) => {
+        const configUid = getPageUidByPageTitle(CONFIG);
+        //
+        // How can this be optimized?
+        //
+        const attributesNode = getSubTree({
+          key: "attributes",
+          parentUid: configUid,
+        });
+        const attributeUid = getSubTree({
+          key: attributeName,
+          parentUid: attributesNode.uid,
+        }).uid;
+        const options = getSubTree({
+          key: "options",
+          parentUid: attributeUid,
+        }).children.map((t) => t.text);
+        renderOverlay({
+          Overlay: ChooseAttributeOverlay,
+          id: "attribute-select",
+          props: { options, uid, attributeName },
+        });
+      }}
+    />
+  );
+};
+
+const renderAttributeButton = (
+  parent: HTMLSpanElement,
+  attributeName: string,
+  blockUid: string
+) => {
+  const containerSpan = document.createElement("span");
+  containerSpan.onmousedown = (e) => e.stopPropagation();
+  ReactDOM.render(
+    <AttributeButton attributeName={attributeName} uid={blockUid} />,
+    containerSpan
+  );
+  parent.appendChild(containerSpan);
+};
+
+const AttributeConfigPanel = () => {
+  const [value, setValue] = useState("");
+  const [definedAttributes, setDefinedAttributes] = useState<string[]>(
+    getDefinedAttributes()
+  );
+  const [activeTab, setActiveTab] = useState(definedAttributes[0]);
+  const handleTabChange = (tabName: string) => {
+    setActiveTab(tabName);
+  };
+  const handleRemoveAttribute = (attribute: string) => {
+    setDefinedAttributes((attrs) => attrs.filter((a) => a !== attribute));
+    const index = definedAttributes.indexOf(activeTab);
+    if (attribute === definedAttributes[index]) {
+      setActiveTab(definedAttributes[0]);
+    }
+  };
+  const configUid = getPageUidByPageTitle(CONFIG);
+  const attributesUid = getSubTree({
+    key: "attributes",
+    parentUid: configUid,
+  }).uid;
+  window.roamAlphaAPI.data.block.update({
+    block: {
+      uid: attributesUid,
+      open: false,
+    },
+  });
+  const attributesInGraph = (
+    window.roamAlphaAPI.data.fast.q(
+      `[:find
+        (pull ?page [:node/title])
+      :where
+        [?b :attrs/lookup _]
+        [?b :entity/attrs ?a]
+        [(untuple ?a) [[?c ?d]]]
+        [(get ?d :value) ?s]
+        [(untuple ?s) [?e ?uid]]
+        [?page :block/uid ?uid]
+      ]`
+    ) as [PullBlock][]
+  )
+    .map((p) => p[0]?.[":node/title"] || "")
+    .filter((a) => !definedAttributes.includes(a));
+
+  return (
+    <div className={`${Classes.DIALOG_BODY} m-0`}>
+      <div className="flex mb-8">
+        <div id="attribute-select-autocomplete">
+          <AutocompleteInput
+            value={""}
```

Comment:
`onConfirm={() => setValue("")}`

Reply:
Didn't work.  I'm going to look into extending <MenuItemSelect>.
I think it will fit this purpose as well as the inline attribute button.

https://github.com/RoamJS/workbench/assets/3792666/b1bd8822-222a-4611-9408-823ec998f6ea

Reply:
https://github.com/RoamJS/workbench/pull/443/commits/7a5bcb8abf23872e8385c0cfb073f3105fd682cf